src/modules/fw_wind_estimator/FixedwingWindEstimator.hpp

```diff
@@ -0,0 +1,156 @@
+#include <lib/slew_rate/SlewRate.hpp>
+#include <matrix/math.hpp>
+#include <px4_platform_common/px4_config.h>
+#include <px4_platform_common/defines.h>
+#include <px4_platform_common/module.h>
+#include <px4_platform_common/module_params.h>
+#include <px4_platform_common/posix.h>
+#include <px4_platform_common/tasks.h>
+#include <px4_platform_common/px4_work_queue/ScheduledWorkItem.hpp>
+#include <uORB/Publication.hpp>
+#include <uORB/PublicationMulti.hpp>
+#include <uORB/Subscription.hpp>
+#include <uORB/SubscriptionMultiArray.hpp>
+#include <uORB/SubscriptionCallback.hpp>
+#include <uORB/topics/airflow.h>
+#include <uORB/topics/airspeed_validated.h>
+#include <uORB/topics/parameter_update.h>
+#include <uORB/topics/vehicle_angular_velocity.h>
+#include <uORB/topics/vehicle_control_mode.h>
+#include <uORB/topics/vehicle_land_detected.h>
+#include <uORB/topics/vehicle_status.h>
+#include <uORB/topics/vehicle_attitude.h>
+#include <uORB/topics/vehicle_acceleration.h>
+#include <uORB/topics/vehicle_local_position.h>
+
+using matrix::Eulerf;
+using matrix::Quatf;
+
+using uORB::SubscriptionData;
+
+using namespace time_literals;
+
+class FixedwingWindEstimator final : public ModuleBase<FixedwingWindEstimator>, public ModuleParams,
+	public px4::ScheduledWorkItem
+{
+public:
+	FixedwingWindEstimator(bool vtol = false);
+	~FixedwingWindEstimator() override;
+
+	/** @see ModuleBase */
+	static int task_spawn(int argc, char *argv[]);
+
+	/** @see ModuleBase */
+	static int custom_command(int argc, char *argv[]);
+
+	/** @see ModuleBase */
+	static int print_usage(const char *reason = nullptr);
+
+	bool init();
+
+private:
+	void Run() override;
+
+	uORB::SubscriptionCallbackWorkItem _vehicle_angular_velocity_sub{this, ORB_ID(vehicle_angular_velocity)};
+
+	uORB::SubscriptionInterval _parameter_update_sub{ORB_ID(parameter_update), 1_s};
+
+	uORB::Subscription _vehicle_acceleration_sub{ORB_ID(vehicle_acceleration)};     	// linear acceleration
+	uORB::Subscription _vehicle_control_mode_sub{ORB_ID(vehicle_control_mode)};
+	uORB::Subscription _vehicle_land_detected_sub{ORB_ID(vehicle_land_detected)};
+	uORB::Subscription _vehicle_status_sub{ORB_ID(vehicle_status)};
+	uORB::Subscription _vehicle_rates_sub{ORB_ID(vehicle_angular_velocity)};
+	uORB::Subscription _vehicle_attitude_sub{ORB_ID(vehicle_attitude)};
+	uORB::Subscription _vehicle_local_position_sub{ORB_ID(vehicle_local_position)};
+	uORB::Subscription _airspeed_validated_sub{ORB_ID(airspeed_validated)};
+
+	uORB::Publication<airflow_s> _airflow_pub{ORB_ID(airflow)};
+
+	vehicle_control_mode_s			_vcontrol_mode{};
+	vehicle_status_s			_vehicle_status{};
+	vehicle_attitude_s 			_vehicle_attitude{};
+
+	perf_counter_t _loop_perf;
+
+	hrt_abstime _last_run{0};
+
+	float _calibrated_airspeed{0.0f};
+	float _true_airspeed{15.0f};
+	matrix::Quatf 				_attitude{};
+	matrix::Vector3f	_acceleration{0.f, 0.f, 0.f};
+	matrix::Vector3f _local_velocity{0.f, 0.f, 0.f};
+	matrix::Vector3f _gravity{0.f, 0.f, 9.81f};
```

Comment:
You can have this constant in the geo library ([CONSTANTS_ONE_G](https://github.com/PX4/PX4-Autopilot/blob/0cefd74fee03c4ede891daf640bd6a776b23e620/src/lib/geo/geo.h#L54)) and directly use it in the code instead of keeping as a member.